src/relay/backend/build_module.cc

```diff
@@ -397,7 +397,18 @@ class RelayBuildModule : public runtime::ModuleNode {
     relay_module = transform::InferType()(relay_module);
     relay_module = transform::LabelOps()(relay_module);
     relay_module = transform::AnnotateMemoryScope(config_)(relay_module);
-
+    pass_seqs = GetPassPrefix(
```

Comment:
Need to remove all changes in build_module.cc, in other case some cpp/gpu.tests will fail

Reply:
Done